engine/factomParams.go

```diff
@@ -110,7 +112,9 @@ func (f *FactomParams) Init() {
 	f.Sim_Stdin = true
 	f.exposeProfiling = false
 	f.Sync2 = -1
-	f.DebugConsole = false
+	f.DebugConsole = "foobar" //TODO: pretty sure this value is overridden by the default in the flag -- clay
```

Comment:
It is overwritten by the default in the ParseCmdLine. We should remove this comment. As to why we set defaults here? Can't say for certain...